### PR TITLE
feat: Added BTC export Private key test

### PR DESCRIPTION
**Pull request by beemi**
# Linear Ticket
LIQ-23
https://linear.app/liquality/issue/LIQ-23/develop-ability-to-export-private-keys-for-a-given-account

### CODE REVIEW

**Review by b0b4456 — APPROVED** [review]
👍

File: tests/e2e/19_private_keys.spec.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    await page.waitForSelector('#i_have_privacy_button', { visible: true })
+    await page.click('#i_have_privacy_button')
+    await page.waitForSelector('#password', { visible: true })
+    await page.type('#password', password)
+    await page.click('#checkbox')
+    await page.click('#continue_button_to_see_seed_phrase')
+    // Private key screen
+    await page.waitForSelector('#private-key-textarea', { visible: true })
+    const privateKeyTextArea = await page.$eval('#private-key-textarea', el => el.getAttribute('readonly'))
+    expect(privateKeyTextArea).equals('readonly')
```
Comment (b0b4456) [inline tests/e2e/19_private_keys.spec.js:59]: maybe also expect that the textarea contains a string that looks like a privatekey, eg something other than `n/a` ?

Comment (beemi) [reply tests/e2e/19_private_keys.spec.js:59]: yes trying to get the textContent but its empty
image

Comment (beemi) [reply tests/e2e/19_private_keys.spec.js:60]: ` const privateKey = await page.$eval('#private-key-textarea', el => el.value)
    expect(privateKeyTextArea).equals('readonly')
    expect(privateKey, 'Private key export value shouldn\'t be n/a')
      .to.not.contain.oneOf(['n/a', 'NaN', null])`

Comment (beemi) [reply tests/e2e/19_private_keys.spec.js:60]: image

**Pull request merged**